src/sdl2_compat.c

```diff
@@ -3914,6 +3914,229 @@ DECLSPEC void SDLCALL SDL_SIMDFree(void *ptr)
 }
 
 
+
+static SDL_bool SDL_IsSupportedAudioFormat(const SDL_AudioFormat fmt)
+{
+    switch (fmt) {
+    case AUDIO_U8:
+    case AUDIO_S8:
+    case AUDIO_U16LSB:
+    case AUDIO_S16LSB:
+    case AUDIO_U16MSB:
+    case AUDIO_S16MSB:
+    case AUDIO_S32LSB:
+    case AUDIO_S32MSB:
+    case AUDIO_F32LSB:
+    case AUDIO_F32MSB:
+        return SDL_TRUE; /* supported. */
+
+    default:
+        break;
+    }
+
+    return SDL_FALSE; /* unsupported. */
+}
+
+static SDL_bool SDL_IsSupportedChannelCount(const int channels)
+{
+    return ((channels >= 1) && (channels <= 8)) ? SDL_TRUE : SDL_FALSE;
+}
+
+
+typedef struct {
+    SDL_AudioFormat src_format;
+    Uint8 src_channels;
+    int src_rate;
+    SDL_AudioFormat dst_format;
+    Uint8 dst_channels;
+    int dst_rate;
+} AudioParam;
+
+
+DECLSPEC int SDLCALL SDL_BuildAudioCVT(SDL_AudioCVT *cvt,
+                                       SDL_AudioFormat src_format,
+                                       Uint8 src_channels,
+                                       int src_rate,
+                                       SDL_AudioFormat dst_format,
+                                       Uint8 dst_channels,
+                                       int dst_rate)
+{
+    /* Sanity check target pointer */
+    if (cvt == NULL) {
+        return SDL_InvalidParamError("cvt");
+    }
+
+    /* Make sure we zero out the audio conversion before error checking */
+    SDL_zerop(cvt);
+
+#if 0
```

Comment:
Should we `#if 0` the `SDL_IsSupportedAudioFormat` and `SDL_IsSupportedChannelCount` procedures too?

Reply:
now this is re-enabled !